src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,98 @@
-export const Pagination = () => {};
```

Comment:
The task description mentions a 'select' element to change the `perPage` value, but it's missing from the code. Consider adding it and update the component accordingly.